src/strategies/python.ts

```diff
@@ -103,20 +103,20 @@ export class Python extends BaseStrategy {
     if (!projectName) {
       logger.warn('No project/component found.');
     } else {
-      updates.push({
-        path: this.addPath(`${projectName}/__init__.py`),
-        createIfMissing: false,
-        updater: new PythonFileWithVersion({
-          version,
-        }),
-      });
-      updates.push({
-        path: this.addPath(`src/${projectName}/__init__.py`),
-        createIfMissing: false,
-        updater: new PythonFileWithVersion({
-          version,
-        }),
-      });
+      [
+        `${projectName.replace('-', '_')}/__init__.py`,
+        `${projectName}/__init__.py`,
+        `src/${projectName.replace('-', '_')}/__init__.py`,
+        `src/${projectName}/__init__.py`,
+      ].forEach(path =>
```

Comment:
I've deduplicated them now by using `flatMap` before going into the `forEach`.

> we should add a test to prevent regressions in the future.

Done as well. Helped me to find a bug, so thanks for the reminder. New to JavaScript and I missed that `replace('-', '_')` only replaces the first instance of a hyphen.